src/app/series/directives/series-feed.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { DistributionModel, SeriesModel, StoryModel, TabService } from '../../shared';
+
+@Component({
+  template: `
+    <section *ngIf="series && podcast">
+      <div class="hint" *ngIf="noStories">
+        You have no published stories in this podcast.
+      </div>
+
+      <section *ngIf="!noStories">
+        <ul *ngIf="privateStories.length">
+          <h4>Private</h4>
+          <li *ngFor="let s of privateStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="private">No publication date set.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="futurePublicStories.length">
+          <h4>Set for future publication</h4>
+          <li *ngFor="let s of futurePublicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="futurePublic">Will be published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="publicStories.length">
+          <h4>Public feed</h4>
+          <li *ngFor="let s of publicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="public">Published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+      </section>
+    </section>
+  `,
+  styleUrls: ['./series-feed.component.css']
+})
+
+export class SeriesFeedComponent implements OnDestroy {
+
+  noStories: boolean;
+  series: SeriesModel;
+  publicStories: StoryModel[] = [];
+  futurePublicStories: StoryModel[] = [];
+  privateStories: StoryModel[] = [];
+  podcast: DistributionModel;
+  tabSub: Subscription;
+
+  constructor(tab: TabService) {
+    this.noStories = false;
+    this.tabSub = tab.model.subscribe((s: SeriesModel) => {
+      this.series = s;
+      this.podcast = s.distributions.filter(dist => dist.kind === 'podcast')[0];
+      this.sortStories();
+    });
+  }
+
+  sortStories() {
+    this.podcast
+        .parent
+        .followItems('prx:stories', { sorts: 'updated_at:desc' })
```

Comment:
I also think there's a problem with paging here - CMS will by default only return the first 10, with links to the next/first/last pages.  So here you may need to do something like (don't quote me here - guessing on some syntax):

```
let total = this.series.doc.total('prx:stories');
if (total > 0) {
  this.series.followItems('prx:stories', {per: total, sorts: 'published_at:desc'}).subscribe(stories => {
    // do stuff here
  });
}
```

You'll have to upgrade the `HalDoc.total()` method to optionally accept a `rel` param, the same way the `count()` method does.  And this could be a potentially long-running query, depending on how many stories there are.  (I tried it in prod on a total=318 series, and it was real slow).  But we can optimize that later!  (Perhaps getting CMS to not embed so many things in the respones).